kalite/distributed/urls.py

```diff
@@ -42,6 +44,9 @@
 # serving it otherwise. Cherrypy is currently serving it through modifications
 # in kalite/django_cherrypy_wsgiserver/cherrypyserver.py
 urlpatterns += patterns('',
+    url(r'^%skhan/(?P<path>.*)$' % settings.CONTENT_URL[1:], 'django.views.static.serve', {
```

Comment:
Try merging 0.14.x into develop locally and see the conflicts. You can abort merging with `git reset --hard HEAD`